rmf_task/include/rmf_task/requests/ChargeBattery.hpp

```diff
@@ -67,7 +64,11 @@ class ChargeBattery : public rmf_task::Request
 
   class Implementation;
 private:
-  ChargeBattery();
+  ChargeBattery(
```

Comment:
We'll want to change this class (and the other request classes) to implement `rmf_task::Request::Description` instead of `Request`.